types/ensDomainStatus.ts

```diff
@@ -0,0 +1,7 @@
+export enum EnsDomainStatus {
```

Comment:
If there is one types folder, why this file holds one enum? 🤔

Reply:
Maybe better to create a domain.ts file inside lib/utils?

Reply:
enum is a data type in typescript

Reply:
So should this folder be called data-types?

Reply:
The folder should be named 'types' and should contain all type-related information, including enums

Reply:
"In a Next.js project, a folder named types is typically used to store TypeScript type definitions, including enums, interfaces, and type aliases. This helps in organizing type-related information and keeping your codebase clean and maintainable."

Reply:
For reference: https://medium.com/@mertenercan/nextjs-13-folder-structure-c3453d780366

<img width="784" alt="image" src="https://github.com/user-attachments/assets/70d0c08c-3807-4a64-a9b6-1dec049d0e03">

Reply:
Perfect